libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/PluggableColumnBarCharts.tsx

```diff
@@ -90,8 +90,9 @@ export class PluggableColumnBarCharts extends PluggableBaseChart {
         source: IInsight,
         event: IDrillEvent,
     ): IDrillEventIntersectionElement[] {
+        const isStackBucket = idMatchBucket(BucketNames.STACK);
         const hasStackByAttributes = bucketsFind(insightBuckets(source), (bucket: IBucket) => {
-            return idMatchBucket(BucketNames.STACK) && !bucketIsEmpty(bucket);
+            return isStackBucket(bucket) && !bucketIsEmpty(bucket);
```

Comment:
small thing: you can achieve this with slightly more convenience

```
const stackBucket = insightBucket(source, BucketNames.Stack);
const hasStackAttributes = stackBucket && !bucketIsEmpty(stackBucket);
```

iirc most functions that allow searching for something by predicate also provide convenience that instead of the predicate itself you can pass the value that is typically searched for (localId, id)

Reply:
looks good, thanks 👍